RoomMate.Domain/Dto/ValidationDto.cs

```diff
@@ -37,5 +37,14 @@ public static  bool IsEmpty(string input)
             }
             return true;
         }
+
+        public static bool IsValidId(int? id)
+        {
+            if (id == 0 || id == null)
```

Comment:
good idea, maybe we can move it to BaseDto?

Reply:
done